input/eg01-eg30/plot_d3ploy.py

```diff
@@ -0,0 +1,103 @@
+"""
+This file produces the plots for one scenario.
+The plots are produced for only one calculation method.
+This is not useful to compare different calculation methods
+but to see the outputs of only one simulation.
+"""
+
+import json
+import re
+import subprocess
+import os
+import sqlite3 as lite
+import copy
+import glob
+import sys
+from matplotlib import pyplot as plt
+import numpy as np
+import pandas as pd
+import d3ploy.tester as tester
+import d3ploy.plotter as plotter
+import collections
+
+direc = os.listdir('./')
+
+# Delete previously generated files
```

Comment:
Is this meant to be commented? delete it?

Reply:
ok